migrations/20210902150815_view_ibc_denom_hash_mapping.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE view_ibc_denom_hash_mapping (
+    id BIGSERIAL,
+    denom VARCHAR NOT NULL UNIQUE,
+    hash VARCHAR NOT NULL,
```

Comment:
Is the `hash`  need to be index as well?

Reply:
Thanks for pointing this out! Yes agree, later we may have requirement to return the denom of a given hash.